spec/swagger/swagger_doc_v2.yaml

```diff
@@ -289,46 +289,6 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v2/patch_people_responses.yaml#/422"
-  "/people/{person_id}/images":
```

Comment:
Moved this to v1 docs (with a few improvements) as it's supported (and identical) across both versions of the API. Not entirely sure how it ended up in the v2 docs.